lib/resorts/kicking horse/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  selector: '.lifts',                // selector for lift information
```

Comment:
you need to customize the selector here.  With a brief look at the page, I think you should be able to find all of the elements with class "rcr-element-box-liftstatus" and then dig the lift name and status out from there

Reply:
This actually might be harder than that. Looks like the Kicking Horse conditions page HTML does not actually have lift status. Instead they fetch it independently through:

https://kickinghorseresort.com/refresh/refresh.php

where it is well hidden among the mundane javascript.

It's not like it cannot be parsed, but it's certainly not the ideal case for your first PR :disappointed:

Reply:
I'll try to find a couple that have sources that look like they work!